src/main/webapp/dev/js/common/questionMsq.es6

```diff
@@ -2,6 +2,36 @@ import {
     ParamsNames,
 } from './const.es6';
 
+function setMaxValForMaxSelectableChoicesInput(questionNum) {
+    const $maxSelectableChoicesCheckbox = $(`#msqEnableMaxSelectableChoices-${questionNum}`);
+    const $generateMsqOptionsCheckbox = $(`#generateMsqOptionsCheckbox-${questionNum}`);
+
+    if ($maxSelectableChoicesCheckbox.prop('checked')) {
+        let maxValue = 2;
```

Comment:
I just wanted to initialize the variable. Will remove it.

> instead of the number of available choices.

The `else` condition just below this code does that.